TWLight/resources/models.py

```diff
@@ -432,4 +432,52 @@ class Meta:
 
     def __unicode__(self):
         # As with Stream, do not return the partner name here.
-        return self.full_name
\ No newline at end of file
+        return self.full_name
+
+
+
+class Suggest(models.Model):
```

Comment:
This is a nitpick, but please call this a Suggestion (noun), rather than Suggest (verb). All of our models represent things not actions.